packages/next/src/schematics/init/init.ts

```diff
@@ -20,10 +20,14 @@ const updateDependencies = addDepsToPackageJson(
 export default function(schema: Schema) {
   return chain([
     setDefaultCollection('@nrwl/next'),
-    addPackageWithInit('@nrwl/jest'),
-    addPackageWithInit('@nrwl/cypress'),
-    addPackageWithInit('@nrwl/web'),
-    addPackageWithInit('@nrwl/react'),
+    schema.unitTestRunner === 'jest'
```

Comment:
Maybe in case of enums, it is better to have resolver style methods, so they take unitTestRunnerType as input and produce rule, that could be more intuitive, when reading and futher extending, becasue we are talking about enum